Installation via npm on macOS failed

```$ npm install ijavascript```

```$  ijs
module.js:472
    throw err;
    ^

Error: Cannot find module 'node-uuid'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/ijavascript/bin/ijavascript.js:45:12)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

--- a/bin/ijavascript.js
+++ b/bin/ijavascript.js
@@ -42,7 +42,7 @@ var path = require("path");
 var spawn = require("child_process").spawn;
 var util = require("util");
 
-var uuid = require("node-uuid");
+var uuid = require("uuid");
 
 var usage = [
     "IJavascript Notebook",
@@ -252,7 +252,7 @@ function parseCommandArgs(context) {
             console.log("ijavascript", context.packageJSON.version);
             console.log("jmp", getPackageVersion("jmp"));
             console.log("nel", getPackageVersion("nel"));
-            console.log("node-uuid", getPackageVersion("node-uuid"));
+            console.log("uuid", getPackageVersion("uuid"));
             process.exit(0);
 
         } else {

--- a/package.json
+++ b/package.json
@@ -27,7 +27,8 @@
         "ijskernel": "lib/kernel.js"
     },
     "dependencies": {
-        "jp-kernel": "0.1.x"
+        "jp-kernel": "0.1.x",
+        "uuid": "^3.0.1"
     },
     "devDependencies": {
         "debug": "latest",

--- a/test/mte/index.js
+++ b/test/mte/index.js
@@ -43,7 +43,7 @@ var fs = require("fs");
 var path = require("path");
 var util = require("util");
 var spawn = require("child_process").spawn;
-var uuid = require("node-uuid");
+var uuid = require("uuid");
 
 var jmp = require("jmp");
 var zmq = jmp.zmq;

